sequencer/finalizer.go

```diff
@@ -20,7 +20,9 @@ import (
 	"github.com/jackc/pgx/v4"
 )
 
-const oneHundred = 100
+const (
+	oneHundred = 100
```

Comment:
I think this constant is not used. In the storePendingTransactions we are using the "100" value directly (with  //nolint:gomnd). Maybe we need to replace the "100" for this oneHundred const (or delete this const as is not used)